eclipse-che/che-plugin-registry
# Provide redirect endpoints for JetBrains-based editors

### What does this PR do?
In favor of the incoming changes in JetBrains-based editors, made by https://github.com/che-incubator/jetbrains-editor-images/pull/121 there is a need to add redirect ports, as it done for che-code and che-theia editors.

Signed-off-by: Vladyslav Zhukovskyi <vzhukovs@redhat.com>

### Screenshot/screencast of this PR
N/A


### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/21141


### How to test this PR?
Open any sample from Devfile registry with the following factory link:
`https://che-host/#https://repository/?che-editor=https://gist.githubusercontent.com/vzhukovs/95337765cbde9b6ed1f252e6e77671a2/raw/ce2508d6cd101d400a4b99cc788933ef5f7eff84/che-editor.yaml`
and check that Endpoints panel contains `code-redirect-*` ports


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
